src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -149,90 +149,61 @@ them, and how to use them to configure your environment.
+
+    -   *image:* Defines which AMI will be deployed on the AWS instances to create. The
+        value to provide is the AWS region together with the unique AMI Id, for example : eu-west-1/ami-bff32ccc. 
+        
+    -   *numberOfInstances:* Total number of AWS instances to create for this infrastructure. 
+    
+    -   *numberOfNodesPerInstance:* Total number of Proactive Nodes to deploy in each AWS created instance. 
+    
 +
-WARNING: Only m1.xlarge and c1.xlarge instances are able to launch
-x86_64 AMI. When trying to run such AMI, if INSTANCE_TYPE is
-not one of the xlarge, it will be forced to m1.xlarge. If you
-want to control the cost of your instances, do not run x86_64
-AMIs.
+TIP: If all the nodes of an AWS instance are removed, the instance will be terminated.
 +
-    -   *MAX_INST:* Sets a maximum number of instances that can be
-        deployed simultaneously on this infrastructure, which is useful
-        to control the cost of instances.
-
-Using the Scheduler and the Resource Manager with an EC2 NodeSource
-requires specific configuration. EC2 instances, indeed, are located on a
-private network, behind a NAT device: this allows fast communications
-between two EC2 instances, but this is an issue when using the RMI
-protocol. The workaround is to use a PAMR router, or the HTTP protocol.
-
-For that reason, the HTTP protocol is forced for communications going
-from the RM to the EC2 nodes. On the other way, from EC2 to the RM,
-using HTTP communications is also the simplest, also RMISSH is possible
-but requires manual configuration for deploying RSA keys. To configure
-your RM with HTTP, all you need to specify is a configuration file with
-the following properties, for both Resource Manager *and* Scheduler.
-
-``` {.xml}
-<prop key="proactive.net.nolocal" value="true"/>
-<prop key="proactive.communication.protocol" value="http" />
-<prop key="proactive.http.port" value="PORT" />
-<prop key="proactive.net.noprivate" value="true" />
-<prop key="proactive.useIPaddress" value="true" />
-```
-
-Note that you need to specify a different port for both configurations.
-Sample configurations can be found in */samples/ec2*.
+    
+    -   *operatingSystem:* The operating system running in the AWS instance.
+    
+    -   *downloadCommand:* The command to download the Proactive node.jar. This command is executed in all the newly created AWS instances at cloud init time. 
+    	The URL needs to be accessible from the AWS cloud.
+    	
+    -   *additionalProperties:* Additional java command properties to be added when connecting the node.jar to the resource manager (e.g. \"-Dpropertyname=propertyvalue\").
+     
+    -   *ram:* The desired RAM amount expressed in Mega Bytes for each AWS instance that needs to be created.
+    
+    -   *cpu:* The desired CPU amount amount expressed in units for each AWS instance that needs to be created.
```

Comment:
What do you mean by _units_? number of CPU cores? number of CPU sockets? or do you mean Amazon EC2 Compute Units (ECU)?

https://aws.amazon.com/ec2/faqs/#What_is_an_EC2_Compute_Unit_and_why_did_you_introduce_it